src/VirtualMonoRepo/source-mappings.json

```diff
@@ -210,11 +215,24 @@
         },
         {
             "name": "winforms",
-            "defaultRemote": "https://github.com/dotnet/winforms"
+            "defaultRemote": "https://github.com/dotnet/winforms",
+            "include": [
+                "src/System.Windows.Forms.Design/src/Resources/colordlg.data",
+                "src/System.Windows.Forms/src/Resources/System/Windows/Forms/Design/Thumbs.db"
+            ],
+            "exclude": [
+                // Non-OSS license - https://github.com/dotnet/winforms/issues/10779
+                "https://github.com/dotnet/winforms/blob/main/src/System.Windows.Forms/tests/IntegrationTests/DesignSurface/LICENSE.txt"
+            ]
         },
         {
             "name": "wpf",
-            "defaultRemote": "https://github.com/dotnet/wpf"
+            "defaultRemote": "https://github.com/dotnet/wpf",
+            "include": [
+                "src/Microsoft.DotNet.Wpf/src/PresentationFramework/**",
+                "src/Microsoft.DotNet.Wpf/src/Shared/Tracing/resources/**",
+                "Microsoft.DotNet.Wpf/src/WpfGfx/core/geometry/Scanner.doc"
+            ]
```

Comment:
Do we still need this now that there is no global exclude anymore?

Reply:
I believe so. From https://github.com/dotnet/source-build/issues/4108 and https://github.com/dotnet/source-build/issues/4087#issuecomment-1930933536, the proposed changes to the binary scanner would require explicit definitions of allowed inclusions. I did remove "Microsoft.DotNet.Wpf/src/WpfGfx/core/geometry/Scanner.doc" however since it's covered by the *.doc* inclusion from allowed-binaries.txt